### DISCUSSION TITLE
Augmented Reality support for Android, iOS and beyond

### DISCUSSION THREAD

**Discussion by SimonDarksideJ** [stride3d/stride#1624] [Q&A]
I could not find any specific answer and considering Stride is using Xamarin (which supports AR) it may be possible.

Could anyone please tell me if AR support is available through Stride?  with features such as:

* Web Camera access for AR view
* Plane tracking through ARKit/ARCore 
* Camera frame access or direct access

Ultimately, could stride be used in an AR context with handheld or head-worn AR devices?

**Comment by vpenades** [comment]
I'm also interested in this.

My understanding is that most standalone headsets running on Android use OpenXR as the main API to handle Augmented Reality.

Luckily, Silk has bindings for it: https://github.com/dotnet/Silk.NET/tree/main/src/OpenXR
